src/components/single_use_components/ProfileListOptions.js

```diff
@@ -10,8 +10,11 @@ import {
 } from "../../helpers/modalFactory";
 import { handleSignOut } from "../../helpers/signOut";
 import { MOCK_USER } from "../../constants/MockUser";
+import AlgoquantApiContext from "../../constants/ApiContext";
 
 export default function ProfileListOptions(props, { navigation }) {
```

Comment:
is the navigation prop being used? i see it on vscode not being used. @WillOrtiz7

Reply:
> is the navigation prop being used? i see it on vscode not being used. @WillOrtiz7

It will eventually be used for the trade history table since we need to navigate from the profile screen to the screen containing the trade history table

Reply:
nice ok